Mathlib/Algebra/Algebra/Subalgebra/Basic.lean

```diff
@@ -618,6 +618,10 @@ def adjoin (s : Set A) : Subalgebra R A :=
   { Subsemiring.closure (Set.range (algebraMap R A) ∪ s) with
     algebraMap_mem' := fun r => Subsemiring.subset_closure <| Or.inl ⟨r, rfl⟩ }
 
+@[simp]
+theorem adjoin_toSubsemiring (s : Set A) :
```

Comment:
Might be nicer to get this by tagging `adjoin` with `simps`

Reply:
Good idea!